src/examples/sendBackrunBundle.ts

```diff
@@ -43,41 +44,61 @@ const handleBackrun = async (
     provider: JsonRpcProvider,
     matchmaker: Matchmaker,
     pendingMutex: Mutex,
-) => {
-    console.log("pending tx", pendingTx)
-    const targetBlock = await provider.getBlockNumber() + 2
+    pendingTxHashes: AsyncArray<string>,
+): Promise<void> => {
+    pendingTx.mevGasPrice
+    if (!pendingTxHashes.includes(pendingTx.hash)) {
```

Comment:
This is a promise, needs await?

Reply:
oh I see, I was looking at the following line. thanks, fixed